kits/arm/ex_AR_kit.py

```diff
@@ -58,6 +58,29 @@
 # # Target variables
 # target_joints = np.zeros(arm.size)
 
+# Print Instructions
+instructions = (
+    "Mode: {}\n"
```

Comment:
I don't think there is a reason to include the run mode here.  The instructions don't dynamically update, so this is always going to list the startup mode, which is misleading...

Reply:
(although we probably should consolidate the logic between here and the `AR_kit_w_gripper`, where these values are used, update the instructions, and update the display of the instructions on the device)